microsetta_admin/templates/email_stats_pulldown.html

```diff
@@ -2,7 +2,8 @@
 {% block head %}
 <link rel="stylesheet" type="text/css" href="/static/vendor/DataTables/DataTables-1.10.21/css/jquery.dataTables.css">
 <script src="/static/vendor/DataTables/DataTables-1.10.21/js/jquery.dataTables.js"></script>
-
+<script src="https://cdn.datatables.net/buttons/1.6.5/js/dataTables.buttons.min.js"></script>
+<script src="https://cdn.datatables.net/buttons/1.6.5/js/buttons.html5.min.js"></script>
```

Comment:
Why the version mismatch?  Can use what is in /static/vendor/DataTables?

Reply:
I don't think its necessarily a version mismatch. The official example uses 1.6.5 for the buttons piece, and 1.10.22 for the jquery datatable itself. But good call, didn't realize we already had a version local!